base/setup/reactos/lang/de-DE.rc

```diff
@@ -53,27 +53,30 @@ CAPTION "ReactOS-Setup"
 FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL "", IDC_PARTITION, "SysTreeList32", WS_BORDER | WS_VISIBLE | WS_TABSTOP | LVS_REPORT | LVS_SINGLESEL, 7, 7, 303, 112
-    PUSHBUTTON "&Erstellen", IDC_PARTCREATE, 7, 122, 50, 15
-    PUSHBUTTON "&Löschen", IDC_PARTDELETE, 63, 122, 50, 15
-    PUSHBUTTON "&Treiber", IDC_DEVICEDRIVER, 174, 122, 50, 15, WS_DISABLED
-    PUSHBUTTON "E&rweiterte Optionen...", IDC_PARTMOREOPTS, 230, 122, 80, 15
+    PUSHBUTTON "&Initialize", IDC_INITDISK, 7, 122, 50, 14 // NOTE: At same position as the next button!
+    PUSHBUTTON "&Erstellen", IDC_PARTCREATE, 7, 122, 50, 14
+    PUSHBUTTON "&Löschen", IDC_PARTDELETE, 63, 122, 50, 14
+    PUSHBUTTON "&Treiber", IDC_DEVICEDRIVER, 174, 122, 50, 14, WS_DISABLED
+    PUSHBUTTON "E&rweiterte Optionen...", IDC_PARTMOREOPTS, 230, 122, 80, 14
     // LTEXT "Klicken Sie auf Weiter um die Zusammenfassung zu überprüfen.", IDC_STATIC, 10, 180, 277, 20
 END
 
-IDD_PARTITION DIALOGEX 0, 0, 145, 90
+IDD_PARTITION DIALOGEX 0, 0, 200, 120
 STYLE DS_SHELLFONT | WS_VISIBLE | WS_CAPTION
-CAPTION "Erstelle Partition"
+CAPTION "Erstelle und formatiere Partition"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    CONTROL "", IDC_UPDOWN1, "msctls_updown32", WS_VISIBLE, 104, 22, 9, 13
-    CONTROL "Erstelle und formatiere Partition", IDC_STATIC, "Button", BS_GROUPBOX, 7, 5, 129, 57
-    LTEXT "Größe:", IDC_STATIC, 13, 24, 27, 9
-    EDITTEXT IDC_PARTSIZE, 58, 22, 47, 13, WS_VISIBLE | WS_TABSTOP
-    LTEXT "GB", IDC_UNIT, 117, 24, 14, 9
-    LTEXT "Dateisystem:", IDC_STATIC, 13, 46, 42, 9
-    CONTROL "", IDC_FSTYPE, "ComboBox", WS_VISIBLE | WS_TABSTOP | CBS_DROPDOWNLIST, 58, 42, 73, 50
-    PUSHBUTTON "&OK", IDOK, 35, 68, 47, 15, WS_VISIBLE | WS_TABSTOP
-    PUSHBUTTON "&Abbrechen", IDCANCEL, 87, 68, 47, 15, WS_VISIBLE | WS_TABSTOP
+    LTEXT "&Größe:", IDC_STATIC, 7, 9, 70, 9
+    EDITTEXT IDC_EDIT_PARTSIZE, 82, 7, 47, 13, ES_RIGHT | ES_NUMBER | WS_GROUP
+    CONTROL "", IDC_UPDOWN_PARTSIZE, UPDOWN_CLASS, UDS_SETBUDDYINT | UDS_ALIGNRIGHT |
+            UDS_AUTOBUDDY | UDS_ARROWKEYS | WS_GROUP, 120, 22, 9, 13
+    LTEXT "MB", IDC_UNIT, 134, 9, 14, 9
+    AUTOCHECKBOX "&Extended partition (MBR disks only)", IDC_CHECK_MBREXTPART, 7, 22, 180, 14
+    LTEXT "&Dateisystem:", IDC_FS_STATIC, 7, 46, 70, 9
+    COMBOBOX IDC_FSTYPE, 82, 44, 100, 50, WS_VISIBLE | WS_TABSTOP | CBS_DROPDOWNLIST
+    AUTOCHECKBOX "&Quick format", IDC_CHECK_QUICKFMT, 7, 59, 180, 14
+    PUSHBUTTON "&OK", IDOK, 88, 98, 50, 14, WS_VISIBLE | WS_TABSTOP
+    PUSHBUTTON "&Abbrechen", IDCANCEL, 143, 98, 50, 14, WS_VISIBLE | WS_TABSTOP
```

Comment:
IDOK and IDCANCEL should usually not have accelerators. Have a look at the MS GUI pdf that you recently sent me. I can live with fixing that in a future PR ofc.

Reply:
fixed